docs/pages/performance/table-emotion.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
-// import { experimentalStyled as styled } from '@material-ui/core/styles';
+// import { styled } from '@material-ui/core/styles';
```

Comment:
I think it would be useful to leave some comment in the code for new team member like me 😂

Reply:
I don't remember why it was commented :D It was changed by the replace. But agree 100% :)